src/main/osd/osd.c

```diff
@@ -496,10 +496,18 @@ static void osdCompleteInitialization(void)
     // Display betaflight logo
     osdDrawLogo(midCol - (OSD_LOGO_COLS) / 2, midRow - 5, DISPLAYPORT_SEVERITY_NORMAL);
 
-    char string_buffer[30];
-    tfp_sprintf(string_buffer, "V%s", FC_VERSION_STRING);
-    displayWrite(osdDisplayPort, midCol + 5, midRow, DISPLAYPORT_SEVERITY_NORMAL, string_buffer);
-#ifdef USE_CMS
+    // sizeof includes the terminating NULL so add space for the V
+    char version_str_buf[sizeof(FC_VERSION_STRING) + 1];
+    tfp_sprintf(version_str_buf, "V%s", FC_VERSION_STRING);
+    size_t version_str_len = strlen(version_str_buf) + 1;
```

Comment:
```suggestion
    size_t version_str_len = strlen(version_str_buf);
```

Reply:
No, `strlen()` returns the number of characters in the version string, excluding the terminating NULL, so we do convert all the characters to upper case, allowing for the extra `V` at the start.

Reply:
Thanks missed that.

Reply:
@SteveCEvans : version_str_buf already has V included